pkg/certgen/scanner_test.go

```diff
@@ -0,0 +1,82 @@
+package certgen
+
+import (
+	"crypto/tls"
+	"testing"
+
+	"github.com/stackrox/rox/pkg/mtls"
+	testutilsMTLS "github.com/stackrox/rox/pkg/mtls/testutils"
+	"github.com/stackrox/rox/pkg/x509utils"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestIssueScannerCerts(t *testing.T) {
+	suite.Run(t, new(issueScannerCertTestSuite))
+}
+
+type issueScannerCertTestSuite struct {
+	suite.Suite
+}
+
+func (s *issueScannerCertTestSuite) SetupTest() {
+	err := testutilsMTLS.LoadTestMTLSCerts(s.T())
+	s.Require().NoError(err)
+}
+
+func (s *issueScannerCertTestSuite) TestIssueScannerCertsSAN() {
+	type testCase struct {
+		name          string
+		namespace     string
+		scannerSANs   []string
+		scannerDBSANs []string
+	}
+
+	cases := []testCase{
```

Comment:
nit: if you like it you can define the struct inside the `cases` definition.
See: https://go.dev/play/p/dZ1-OlUkMC7

Reply:
Done.